numba/tests/support.py

```diff
@@ -296,6 +307,19 @@ def _assertNumberEqual(first, second, delta=None):
             # Not floats.
             pass
 
+        # if absolute comparison is set, use it
+        if abs_tol is not None:
+            if abs_tol == "eps":
+                rtol = np.finfo(type(first)).eps
+            elif isinstance(abs_tol, float):
+                rtol = abs_tol
+            else:
+                self.fail(
```

Comment:
How about raising a ValueError instead? This is not a test failure, this is bug in a test.